src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -95,6 +90,28 @@ bool AudioStreamAAudio::isSupported() {
     return openResult == 0;
 }
 
+// Static 'C' wrapper for the error callback method.
+// Launch a thread to handle the error.
+// That other thread can safely stop, close and delete the stream.
+void AudioStreamAAudio::internalErrorCallback(
+        AAudioStream *stream,
+        void *userData,
+        aaudio_result_t error) {
+    AudioStreamAAudio *oboeStream = reinterpret_cast<AudioStreamAAudio*>(userData);
+    if (oboeStream == nullptr) {
```

Comment:
Remove NULL check.
Add comment about error callbacks not getting called after stopped and closed.